comparison.md

```diff
@@ -102,7 +115,8 @@ the overall size of each ecosystem.
 ### Platform support (Android, GWT)
 
 AssertJ supports Android (though I had to use 2.x because the dexer rejected
-3.x, even when I used only `Java6Assertions`).
+3.x, even when I used only `Java6Assertions`). Possibly this was an issue with
+my build setup.
```

Comment:
This paragraph is also written in the first person, which I think is another potential cause for confusion.

Could this be re-phrased to be in the third-person? Maybe something like the following?

> AssertJ supports Android, but note that you may be forced to use 2.x instead of 3.x, even if you only use `Java6Assertions`, since at time of writing the dexer has problem(s) with 3.x.